config/default.json

```diff
@@ -5,6 +5,7 @@
 	"allowExtends": false,
 	"required": [
 		"README.md",
-		"package.json"
+		"package.json",
+		".npmignore"
```

Comment:
this will instantly break every single component. I don't want to do this right now. it needs much more thought

Reply:
We could manually add .gitignore to the preexisting packages 🤷‍♂️ 
I'll remove this code tomorrow. Let me know what you come up with 😸